01-Python/01-python.md

```diff
@@ -5,9 +5,7 @@
     1. [Zen of Python](https://en.wikipedia.org/wiki/Zen_of_Python)
     2. [Introduction to Python Programming](https://click.linksynergy.com/deeplink?id=vedj0cWlu2Y&mid=39197&u1=ddfreepython2&murl=https%3A%2F%2Fwww.udemy.com%2Fcourse%2Fpythonforbeginnersintro%2F)
     3. [PEP8](https://realpython.com/python-pep8/)
-    4. [google pyguide](https://google.github.io/styleguide/pyguide.html)
     5. [OOP and python](https://realpython.com/inheritance-composition-python/#whats-inheritance)
-    6. [Build cli - click](https://pymbook.readthedocs.io/en/latest/click.html)
```

Comment:
I think this might be useful just as an FYI, building CLIs is often quite useful, for example, the ai services releases automation was built with click

Reply:
They can learn it if they need to build a CLI, but it doesn't have to be in the chapter